### PR TITLE
Added one step in install process due to UI changes

### PR DESCRIPTION
**Pull request by CherylFrankenfield**
### Tell us why

The UI changed and there is a slightly different step-through process for installation. 

### Anything else you'd like to share?

It's likely not necessary, though adding a screenshot of the view may be useful. 

### Are you making a change to site code?

If you're changing site code (rather than the content of a doc), please follow
[conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/) 
in your commit messages and pull request title.

### CODE REVIEW

File: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx [hunk 1]
```diff
@@ -98,19 +98,19 @@ No matter which option you use to deploy Browser, the end result is to inject th
...
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com) > Browser > <Icon name="fe-plus-circle"/>
-   + Add More Data**.
-2. Find the header "Back-end, front-end, and mobile applications" and select the New Relic Browser tile. A menu with account options will appear and prompt you to select the account you want to add this instrumentation to. Choose your account and click the continue button.
-4. From the Browser deployment page, select **Enable via APM**.
-5. Select your choice of available options for **Browser agent instrumentation** [features](/docs/browser/new-relic-browser/getting-started/new-relic-browser).
-6. Type or use the search window to find the app's name.
-7. Select **Enable**.
+1. Go to **[one.newrelic.com](https://one.newrelic.com)**, select **Browser**, and then select **Add more data**.
+2. In the **Back-end, front-end, and mobile applications** section, select the **New Relic Browser** tile. When prompted to select the account you want to add this instrumentation to, choose your account, and click **Continue**.
```
Comment (mmfred) [inline src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx:102]: @CherylFrankenfield - I did some further revising, following along in the UI. Let me know if I went off down a wrong road.

File: src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx [hunk 2]
```diff
@@ -98,19 +98,19 @@ No matter which option you use to deploy Browser, the end result is to inject th
...
+4. From the **Get started with New Relic Browser page**, in the **Choose a deployment method** section, select **Enable via New Relic APM**.
+5. In the **Choose your instrumentation** section, select **Lite**, **Pro**, or **Pro + SPA**. See [features](/docs/browser/new-relic-browser/getting-started/new-relic-browser) for details about each .
```
Comment (mmfred) [inline src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx:104]: I'm not sure that the link correctly supports the selections in this section. If you're okay with the changes, but the link is wrong, let's find a better one.

**Review by mmfred — CHANGES_REQUESTED** [review]
Hi @CherylFrankenfield I made some edits and left some comments for you to review in my commit. Please take a look. Thank you!

**Review by mmfred — APPROVED** [review]

**Comment by urbiz-nr** [comment]
Checked locally, looks great! You can merge, @mmfred !

**Pull request merged**